### PR TITLE
Release/nft approval

### PR DESCRIPTION
**Pull request by Ben-Rey**
I added approval functions in the wrapper and updated the version on package.json for the new release.

### CODE REVIEW

File: smart-contracts/assembly/contracts/NFT/NFTWrapper.ts [hunk 1]
```diff
@@ -101,4 +101,47 @@ export class NFTWrapper {
...
+   * @param tokenId - Token ID
+   * @param addressFrom - address of the owner
+   * @param addressTo - address of the recipient
+   * @param tokenId - Token ID
+   *
+   */
+  transferFrom(tokenId: u64, addressFrom: string, addressTo: string): void {
+    call(
+      this._origin,
+      'approve',
```
Comment (gregLibert) [inline smart-contracts/assembly/contracts/NFT/NFTWrapper.ts:128]: same function called with different arguments. Is it normal ?

**Review by gregLibert — COMMENTED** [review]
:question:

File: smart-contracts/assembly/contracts/NFT/NFTWrapper.ts (hunk 1 above)
Comment (Ben-Rey) [reply smart-contracts/assembly/contracts/NFT/NFTWrapper.ts]: Fixed it thanks !

File: smart-contracts/assembly/contracts/NFT/NFTWrapper.ts [hunk 2]
```diff
@@ -101,4 +101,47 @@ export class NFTWrapper {
...
+   * @param tokenId - Token ID
+   * @param address - address to approve
+   *
+   */
+  approve(tokenId: u64, address: string): void {
+    call(this._origin, 'approve', new Args().add(tokenId).add(address), 0);
+  }
+
+  /**
+   * Transfer a chosen token from the caller to the toAddress.
```
Comment (Thykof) [inline smart-contracts/assembly/contracts/NFT/NFTWrapper.ts:117]: it's not from the caller but from the address given as first argument

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by gregLibert — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Pull request merged**